src/main/java/de/tum/in/www1/artemis/web/rest/CourseResource.java

```diff
@@ -376,6 +376,20 @@ else if (exercise instanceof ProgrammingExercise) {
 
             long numberOfAssessments = resultService.countNumberOfFinishedAssessmentsForExercise(exercise.getId());
 
+            long numberOfComplaints = complaintRepository.countByResult_Participation_Exercise_IdAndComplaintType(exercise.getId(), ComplaintType.COMPLAINT);
+            long numberOfComplaintResponses = complaintResponseRepository.countByComplaint_Result_Participation_Exercise_Id_AndComplaint_ComplaintType(exercise.getId(),
+                    ComplaintType.COMPLAINT);
+            // TODO: Hanya, maybe complaints about your own assessment (numberOfTutorComplaints) should be taken into consideration
```

Comment:
I agree, we should ignore complaints related to the tutor's assessment and subtract them from the overall number

Reply:
The only reason I did not add it yet and left the comment instead is because `numberOfTutorComplaints` is currently a bit buggy and will be fixed in #1143

Reply:
As discussed before the meeting, we both actually mean the same thing but I guess my Todo was a bit unclear, I will rephrase it. And I also suggest that the property `numberOfTutorComplaints` should be renamed to `nrOfComplaintsAboutOwnAssessment` in #1143 to avoid any misunderstanding